src/main/kotlin/no/nav/syfo/dialogmotesvar/domain/KDialogmotesvar.kt

```diff
@@ -1,8 +1,28 @@
 package no.nav.syfo.dialogmotesvar.domain
 
+import no.nav.syfo.domain.PersonIdent
+import java.time.OffsetDateTime
+import java.util.*
+
 data class KDialogmotesvar(
-    val personident: String,
-    val type: String,
-    val dialogmoteUuid: String,
-    val virksomhetsnummer: String
+    val ident: PersonIdent,
```

Comment:
Dumt spørsmål fra meg, men finnes det en topic med disse objektene eller kommer det senere?

Reply:
Det finnes en topic ja, den eies av isdialogmote :)

Reply:
Ok. For da må vel feltene hete det samme der, eller? 🤔 Siden dere gjør en rename her :)

Reply:
Ah da skjønner jeg. Vi lagde dette k-objektet før vi lagde topicen i isdialogmote, nå er topicen laget, så nå endrer vi her så det blir likt.

Reply:
Skjønner 👍